GOVERNANCE.md

```diff
@@ -0,0 +1,95 @@
+
+### Copyright
+
+For copyright and trademark information, see [Trademark Usage](https://www.linuxfoundation.org/legal/trademark-usage)
+
+## Roles and Responsibilities
+
+OpenFGA uses a Teams concept to divide up roles and responsibilities. Teams are defined in Github.
+
+### Teams
+Currently there exist three main teams:
+
+* Core
+* Product
+* Contributors
+
+#### Core
+
+The @core team is concerned with the codebase and technical implementation details for the product. 
+
+Responsibilities include:
+- review RFCs or delegate their review to a more relevant team
+- reach consensus and usher each RFC through the resolution process
+- collaborate with @community to collect feedback for RFCs
+- ensure RFCs are acceptable to maintain
+- govern OpenFGA repos
+- PR review and merging
+- curating releases. 
+
+Secondary priorities involve collaborating with @product on the roadmap, user research, and iterating on the product experience.
+
+#### Product
+
+The @product team is concerned with iterating on the product itself. They govern the product roadmap, product marketing, design principles, user research and subject matter expertise, and other product concerns.
+
+Responsibilities include:
+- develop and iterate on the governance model
+- review RFCs or delegate their review to a more relevant team
+- maintain OpenFGA's design principles through the RFC review process
+- clarify OpenFGA's design principles over time
+- collaborate with @core to develop a roadmap that fits OpenFGA's design principles
+- collaborate with @core and @contributors on user research
+- iterate on the product experience
+
+#### Contributors
+
+The @contributors team exists to keep track of and grant certain priveliges to anyone who contributes to the product.
+
+### Joining a team
+
+Anyone can become a member of @contributors team: reach out to anyone on the @core or @product teams via any of the channels in the [Support](#support) section with a request to join and they will make the change.
+
+From there, being promoted to being a member of the @core or @product teams when they have a will to be more involved in any of those teams' responsibilities, and have made significant enough contributions to the product that they can get a majority vote from those teams. For more info, see the [Governance-Related Policies and Procedures](#governance-related-policies-and-procedures) section.
+
+### Leaving a team
+
+Should you wish to leave a team, reach out to a member of the Core team via any of the channels in the [Support](#support) section with a request.
+
+### Creating a new 
+
+As the product grows, it will become more useful to further subdivide. This will enable the project to assign code coverage, streamline conversations, and create subject-matter-expertise where possible. If you have an idea for a new team, start a [GitHub Discussion](https://github.com/orgs/openfga/discussions) in order for the community to track it and gain alignment.
```

Comment:
```suggestion
As the product grows, it will become more useful to further subdivide the teams. This will enable the project to assign code coverage, streamline conversations, and create subject-matter-expertise where possible. If you have an idea for a new team, start a [GitHub Discussion](https://github.com/orgs/openfga/discussions) in order for the community to track it and gain alignment.
```